### ISSUE TITLE
How to get correct 'train_val_labels.pkl' ?

### ISSUE THREAD

**Issue by LiuyhLinda** [jackyjsy/CVPR21Chal-SLR#23]
Thanks for your project.
 I'm very interested in the network you designed. But when goes to 'test_joint.yaml', I can not get the 'train_val_labels.pkl'. The 'train_val_labels.pkl' I merged from 'train_label.pkl' and 'val_gt.pkl' is wrong.
The code is following:
###
import pickle
with open('/media/lyh/data/AUTSL/Autsl/skeleton_data/train_label.pkl', 'rb') as fr_1:
     data_1 = pickle.load(fr_1) 
with open('/media/lyh/data/AUTSL/Autsl/skeleton_data/val_gt.pkl', 'rb') as fr_2:
     data_2 = pickle.load(fr_2)
data = [data_1, data_2]
with open('/media/lyh/data/AUTSL/Autsl/skeleton_data/train_val_labels.pkl', 'wb') as fr_3:
     pickle.dump(data, fr_3)  
​###
Would you mind help me to get correct 'train_val_labels.pkl'?
 I am looking forward to your reply. Thanks in advance.

**Comment by jackyjsy** [comment]
Hi there, the train_val_labels.pkl is only used when attending the challenges where we were allowed to finetune our model on validation set. When you test the model using test_joint.yaml, it only specifies the training loader, so not used in testing. If you only want to test on the test set, you can change it to train_labels.pkl, which won't affects your model accuracy on the test set. I have made a new [commit](https://github.com/jackyjsy/CVPR21Chal-SLR/commit/de6c53a6fbbc08a5d9a728e33614a4541836714c) address this issue.

**Comment by LiuyhLinda** [comment]
It is working! Thanks a lot!
